testsuite/constraint_shape_based_cylinder.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import division, print_function
```

Comment:
shape based constraints are tested in ```constraint_shape_based.py```. This and the ellipsoid test file should be removed.